components/nimbus/src/nimbus.udl

```diff
@@ -76,10 +76,10 @@ enum NimbusError {
     "InvalidPersistedData", "RkvError", "IOError",
     "JSONError", "EvaluationError", "InvalidExpression", "InvalidFraction",
     "TryFromSliceError", "EmptyRatiosError", "OutOfBoundsError","UrlParsingError",
-    "RequestError", "ResponseError", "UuidError", "InvalidExperimentFormat",
```

Comment:
This **may** be a breaking change depending on if the consuming apps depend on those errors. Which is totally okay, but lets note that in our changelog! (or if we can easily verify this definitely isn't a breaking change, just noting tha apps can no longer depend on them is good enough!)

Reply:
This is a good point, I see that Fenix treats `RequestError` and `ResponseError` as non-reportable, so although a breaking change I don't think they are necessary there at least. Is the iOS app the only other place I should be checking?

Reply:
Yup! iOS and Fenix are the only consumers (right now!) - given Fenix depends on it, let's flag this change as a breaking change in the https://github.com/mozilla/application-services/blob/main/CHANGES_UNRELEASED.md, so when we do the next a-s upgrade we replace them with new variants